Sources/PenguinStructures/Graphs/AdjacencyList.swift

```diff
@@ -37,7 +37,7 @@ public struct AdjacencyList<IdType: BinaryInteger>: GraphProtocol {
 	}
 
 	// TODO: consider just making it an `IdType` or an `Int`.
-	/// A logical pointer into the AdjacencyList for a vertex.
+	/// A vertex identifier.
```

Comment:
This description is a bit much like the name of the type; it makes me go, "well, duh!"

Consider whether something like this is an improvement:
```suggestion
	/// The name of a vertex in this graph.
```

Ditto for `EdgeID`.